www/js/onboarding/SummaryPage.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+import { View, ScrollView } from 'react-native';
+import { Button, Surface } from 'react-native-paper';
+import { AppContext } from '../App';
+import { onboardingStyles } from './OnboardingStack';
+import StudySummary from './StudySummary';
+import { setSummaryDone } from './onboardingHelper';
+
+const SummaryPage = () => {
+
+  const { t } = useTranslation();
+  const context = useContext(AppContext);
+  const { refreshOnboardingState } = context;
+
+  function next() {
+    setSummaryDone(true);
+    refreshOnboardingState();
+  };
+
+  // summary of the study, followed by 'next' button
+  return (<>
+    <ScrollView contentContainerStyle={{flex: 1}}>
+      <Surface style={[onboardingStyles.page, {flex:1, gap: 16}]}>
+        <StudySummary />
```

Comment:
why do we need to split `StudySummary` and `SummaryPage`? I thought we were not showing the `StudySummary` in the profile page, only the privacy policy.